balancer/weightedtarget/weightedtarget_test.go

```diff
@@ -306,6 +307,24 @@ func (s) TestWeightedTarget(t *testing.T) {
 			t.Fatalf("picker.Pick, got %v, want SubConn=%v", gotSCSt, sc3)
 		}
 	}
+	// Update the Weighted Target Balancer with an empty address list and no
+	// targets. This should cause a Transient Failure State update to the Client
+	// Conn.
+	config4, err := wtbParser.ParseConfig([]byte(`{}`))
+	if err != nil {
+		t.Fatalf("failed to parse balancer config: %v", err)
```

Comment:
Nit: s/failed/Failed/ (and for all error messages added in this test, as part of this PR)

The capitalization rules which apply for error strings don't apply for log messages and test error messages.

Reply:
Switched in all cases.